### ISSUE TITLE
Camera Anchor FOV doesn't match calibrated setting (stuck at 60FOV)

### ISSUE THREAD

**Issue by CuriousNicholi** [Frooxius/NeosPublic#889]
After the recent updates, the Camera Anchor no longer follows FOV calibration setting. I calibrate the camera FOV to match my external video camera(s) at 43.3, and that's been working great until a couple days ago.
Now, even when setting Manual with the focus ring (not as precise as numbers), Camera Anchors always default to 60 FOV (driven). I've tried calibrating through the number field, slider, and focus ring, but none effect Camera Anchors.

Here's a short video reproducing the issue and all the routes I've tried to get around it.
https://youtu.be/aT084GoBgbg

**Comment by Frooxius** [comment]
Does this happen with any camera? I haven't many any changes to the camera anchors themselves, so it's likely something with the streamer camera that's being used.

**Comment by shiftyscales** [comment]
This also occurs with the default streamer camera, @Frooxius. I've just replicated it.

**Comment by Frooxius** [comment]
What is the FieldOfView set to on the actual InteractiveCameraAnchor component? When in the anchor, the camera will use the FOV defined by the anchor.

I've tried replicating it, but if I set the FieldOfView on the InteractiveCameraAnchor it works correctly. Whenever you create camera anchor, it will use the current FOV of the camera. Have you potentially recreated the camera anchor on your avatar from scratch? The default is 60.
